charts/testkube-operator/values.yaml

```diff
@@ -45,8 +45,8 @@ webhook:
   patch:
     enabled: true
     image:
-      repository: k8s.gcr.io/ingress-nginx/kube-webhook-certgen
-      version: v1.3.0
+      repository: dpejcev/kube-webhook-certgen
```

Comment:
would be nice to move to kubeshop org

Reply:
Needs to be moved to kubeshop org definitely, but I am not sure I have all privileges to setup CI and rest